CHANGES

```diff
@@ -3,20 +3,54 @@ Release 1.8 (development)
 
 .. rubric:: Enhancements
 
+* Implement incremental image updates based on block hash indices.
+  This works by adding an index file containing the hashes of each 4kiB image
+  block in the image to the bundle and then using this to check whether a block
+  is available locally during installation.
+  If that's the case, RAUC doesn't need to download this block.
+  Together with streaming, this means that only a small part of the bundle needs
+  to be downloaded as long as the changes are localized.
+  See the :ref:`documentation for details <sec-incremental-updates>`.
+* Add a slot type which provides atomic bootloader updates for SoCs (like the
+  Rockchip RK3568) which search for a valid image at multiple fixed offsets. (by
+  Matthias Fend)
+  See the :ref:`documentation for details <sec-raw-partition-fallback>`.
+* Add a configuration option for additional arguments to pass to ``casync
+  extract``. (by Ludovico de Nittis)
+* Add initial support for desync (an alternative casync implementation). (by
+  Ludovico de Nittis)
+* Add support for a RAUC data-directory on a shared partition.
```

Comment:
Maybe note that this directory is used for the central slot status unless otherwise configured?